src/draw/grouping.typ

```diff
@@ -109,7 +132,7 @@
           message: "No such element '" + elem + "' in elements " + repr(ctx.nodes.keys()))
         named-drawables.push(ctx.nodes.at(elem).drawables)
       } else {
-        for sub in elem {
+        for sub in resolve-body(ctx, elem) {
```

Comment:
I'm not sure this should work as in this situation:
```
intersections("i", {
  line(...)
  line(...)
})
```
`elem` would be the element function which should be passed directly to `process.element`

Reply:
Besides theres an assert requiring at least one element on line 93/116

Reply:
I've removed the function support for on-layer again. If a user want's a context they can call `get-ctx`.